sdk/internal/recording/recording_test.go

```diff
@@ -368,7 +372,148 @@ func (s *recordingTests) TestRecordRequestsAndFailMatchingForMissingRecording()
 }
 
 func (s *recordingTests) TearDownSuite() {
-	// cleanup test files
-	err := os.RemoveAll("recordings")
-	require.Nil(s.T(), err)
+	files, err := filepath.Glob("recordings/**/*.yaml")
```

Comment:
Is there any way to just put the temp data into a single folder and os.RemoveAll() that? Not a huge deal, but would make cleaning up hand simpler as well.

Reply:
This is from the legacy recording framework and will hopefully be removed altogether in the near future